xml/hwbp_registers.xml

```diff
@@ -1014,13 +1030,35 @@
         5, or 6 and XLEN=64. The fields are defined
         above, in \RcsrTextraThirtytwo.
 
+        Byte-granular comparison of the \RcsrScontext to \FcsrTextraSixtyfourSvalue in
+        \RcsrTextraSixtyfour allows \RcsrScontext to be defined to include
+        more than one element of comparison.  For example, software instrumentation
+        can program the \RcsrScontext value to be the concatenation of different ID contexts
+        such as process ID and thread ID.  The user can then program byte compares
+        to include one or more of the contexts in the compare. For \RcsrTextraSixtyfour
+        there are 5 byte-compare mask bits.
+        
+        Byte masking only applies to \RcsrScontext comparison; i.e when \FcsrTextraSixtyfourSselect is 1.
+        
         <field name="mhvalue" bits="63:51" access="WARL" reset="0">
         </field>
         <field name="mhselect" bits="50:48" access="WARL" reset="0">
         </field>
 
-        <field name="0" bits="47:36" access="R" reset="0" />
+        <field name="0" bits="47:41" access="R" reset="0" />
 
+        <field name="sbytemask" bits="40:36" access="WARL" reset="0">
+            When the least significant bit of this field is 1, it causes bits 7:0
+            in the comparison to be ignored, when \FcsrTextraSixtyfourSselect=1.
+            When the next most significant bit of this field is 1, it causes bits 15:8
+            to be ignored in the comparison, when \FcsrTextraSixtyfourSselect=1
+            When the next most significant bit of this field is 1, it causes bits 23:16
+            to be ignored in the comparison, when \FcsrTextraSixtyfourSselect=1
+            When the next most significant bit of this field is 1, it causes bits 31:24
+            to be ignored in the comparison, when \FcsrTextraSixtyfourSselect=1
+            When the next most significant bit of this field is 1, it causes bits 33:32
+            to be ignored in the comparison, when \FcsrTextraSixtyfourSselect=1
```

Comment:
At some point we'll have to find a nicer way to write this, but I don't have any good ideas right now. Probably can merge like this and fix it later.